internal/kldtx/txnprocessor_test.go

```diff
@@ -605,10 +605,13 @@ func TestOnSendTransactionMessageBadNonce(t *testing.T) {
 
 	txnProcessor := NewTxnProcessor(&TxnProcessorConf{}, &kldeth.RPCConf{}).(*txnProcessor)
 	testTxnContext := &testTxnContext{}
+	// Guessing this is a go version behavior change: using `abc` for nonce throws an error in parsing the
```

Comment:
This is a super helpful review comment. However, I'm not sure we want this in the code forever.
Maybe move it into the PR commentary?